ocs_ci/ocs/machine.py

```diff
@@ -173,7 +173,10 @@ def delete_machine_and_check_state_of_new_spinned_machine(machine_name):
 
 
 def create_custom_machineset(
-    role='app', instance_type='m4.xlarge', label='app-scale', zone='a'
+    role='app',
+    instance_type='m4.xlarge',
+    label='app-scale',
+    zone='a',
```

Comment:
I am thinking about add two additional parameters which will be:
labels and taints with default with type list and default value None,

If some of them will be passed to this function then those extra labels, taints will be added to proper lists in machineset which will make this function more generic.

Then from the place where we would like to create those extra infra nodes we will pass those informations from there where we will read information from values like I mentioned `ocs_operator_infra_nodes_to_label` and `ocs_operator_infra_nodes_to_taint` And based on this info we will pass what we really want from that place.

WDYT?

I am nto sure about default label `app-scale` here, this I think should not be applied on infra nodes, please discuss with author of this function and check usage of that. I think it should be None as default and from places where it's really need to be used and intended to use app-sclae label it has to be passed form place where it's used.